src/components/distributions/editor/TextForm/TextInput.js

```diff
@@ -28,6 +28,10 @@ class TextInputEditor extends Component {
     return editorState.getCurrentContent().getPlainText('')
   }
 
+  _onBlur() {
+    this.props.onBlur(this.state.editorState.getCurrentContent().getPlainText(''))
+  }
```

Comment:
Why does this need to pass the value to the parent? The parent element, TextInput, already has that value in state.value.